source/configuration_manual/authentication/ldap_conf_ext_settings.rst

```diff
@@ -0,0 +1,603 @@
+--------------------
+
+- Default: ``no``
+- Values:  :ref:`boolean`
+
+If authentication binding is used, you can save one LDAP request per login
+if users' DN can be specified with a common template. The template can use
+the standard %variables (see :ref:`ldap_conf_ext_setting-user_filter`).
+Note that you can't use any pass_attrs if you use this setting.
+
+If you use this setting, it's a good idea to use a different
+dovecot-ldap.conf.ext for userdb (it can even be a symlink, just as long as
+the filename is different in userdb's args). That way one connection is used
+only for LDAP binds and another connection is used for user lookups.
+Otherwise the binding is changed to the default DN before each user lookup.
+
+Example:
+
+.. code-block:: none
+
+   auth_bind_userdn = cn=%u,ou=people,o=org
+
+
+.. _ldap_conf_ext_setting-base:
+
+``base``
+--------
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+LDAP base. %variables (see :ref:`config_variables`) can be used here.
+
+
+Example:
+
+.. code-block:: none
+
+   base = dc=mail, dc=example, dc=org
+
+
+.. _ldap_conf_ext_setting-blocking:
+
+``blocking``
+------------
+
+- Default: ``no``
+- Values:  :ref:`boolean`
+
+By default all LDAP lookups are performed by the auth master process.
+If blocking=yes, auth worker processes are used to perform the lookups.
+Each auth worker process creates its own LDAP connection so this can
+increase parallelism. With blocking=no the auth master process can
+keep 8 requests pipelined for the LDAP connection, while with blocking=yes
+each connection has a maximum of 1 request running. For small systems the
+blocking=no is sufficient and uses less resources.
+
+
+.. _ldap_conf_ext_setting-debug_level:
+
+``debug_level``
+---------------
+
+- Default: ``0``
+- Values:  :ref:`uint`
+
+LDAP library debug level as specified by LDAP_DEBUG_* in ldap_log.h.
+Value ``-1`` means everything. You may need to recompile OpenLDAP with debugging enabled
+to get enough output.
+
+
+.. _ldap_conf_ext_setting-dn:
+
+``dn``
+------
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+Specify the Distinguished Name (the username used to login to the LDAP server).
+Leave it commented out to bind anonymously (useful with :ref:`ldap_conf_ext_setting-auth_bind` = yes).
+
+Example:
+
+.. code-block:: none
+
+   dn = uid=dov-read,dc=ocn,dc=ad,dc=jp,dc=.
+
+
+.. _ldap_conf_ext_setting-dnpass:
+
+``dnpass``
+------------
+
+- Default: <empty>
+- Values:  :ref:`string`
+
+Password for LDAP server, used if :ref:`ldap_conf_ext_setting-dn` is specified.
+
+
+.. _ldap_conf_ext_setting-default_pass_scheme:
+
+``default_pass_scheme``
+-----------------------
+
+- Default: ``crypt``
+- Values:  :ref:`string`
+
+Default password scheme. "{scheme}" before password overrides this.
+
+See :ref:`authentication-password_schemes` for a list of supported schemes.
+
+
+.. _ldap_conf_ext_setting-deref:
+
+``deref``
+---------
+
+- Default: ``never``
+- Values:  ``never, searching, finding, always``

```

Comment:
I think nicer if each word is quoted separately, i.e.:

```
``never``, ``searching``, ``finding``, ``always``
```

Reply:
fixed